pkg/apis/build/v1alpha1/image_lifecycle.go

```diff
@@ -22,6 +23,17 @@ func (im *Image) BuilderNotFound() duckv1alpha1.Conditions {
 	}
 }
 
+func (im *Image) BuilderNotReady() duckv1alpha1.Conditions {
+	return duckv1alpha1.Conditions{
+		{
+			Type:    duckv1alpha1.ConditionReady,
+			Status:  corev1.ConditionFalse,
+			Reason:  BuilderNotReady,
+			Message: fmt.Sprintf("Builder %s is not ready.", im.Spec.Builder.Name),
+		},
+	}
+}
+
 func (b *Build) BuilderNotFound() duckv1alpha1.Conditions {
```

Comment:
I realize that you didn't add this now, but while we are here can we remove the `build.BuilderNotFound` method?